samples/unit_testing/subscription-manager/unit_tests/test_mgmtgroup_suborchestrator.py

```diff
@@ -0,0 +1,39 @@
+import azure.durable_functions as df
+from unittest import main
+from unittest import TestCase
+from unittest.mock import patch
+from ..MgmtGroupSubOrchestrator import orchestrator_fn
+
+def call_activity_mock_add_sub_to_mgmt_group(activityName: str):
+    assert activityName == "AddSubscriptionToMgmtGroup"
+
+"""
+Test class for MgmtGroupSubOrchestrator Durable orchestrator that kicks off an activity function
+Mocks the DurableOrchestrationContext and checks the sequence of sub-orchestration calls
+"""
+class TestMgmtGroupSubOrchestrator(TestCase):
```

Comment:
Here, have you thought of leveraging what is already in

- [`tests/orchestrator/orchestrator_test_utils.py`](https://github.com/Azure/azure-functions-durable-python/blob/dev/tests/orchestrator/orchestrator_test_utils.py)
- [`tests/test_utils/ContextBuilder.py`](https://github.com/Azure/azure-functions-durable-python/blob/dev/tests/test_utils/ContextBuilder.py)

I also think this classes and methods should be in the main package, and users should be able to import them just with `from azure.durable_functions.testing import ...`. Refactoring what @scgbear and @davidmrdavid did in [`test_sequential_orchestrator.py`](https://github.com/Azure/azure-functions-durable-python/blob/dev/tests/orchestrator/test_sequential_orchestrator.py) for example.